specs/interop/token-bridging.md

```diff
@@ -59,6 +64,8 @@ Notice that ERC20s that do not implement the standard can still be fungible
 using interop message passing
 using a custom bridge or implementing `sendERC20` and `relayERC20` on their own contracts.
 
+An example implementation of the standard is available at [SuperchainERC20.sol](https://github.com/ethereum-optimism/optimism/blob/91099581e7e129921d794b0a8580ef2c5bb445d8/packages/contracts-bedrock/src/L2/SuperchainERC20.sol)
```

Comment:
```suggestion
An example implementation of the standard is available at [SuperchainERC20.sol](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/L2/SuperchainERC20.sol)
```
could we use the develop branch rather than a fixed commit?

Reply:
fixed!